### DISCUSSION TITLE
How to change updateStrategy

### DISCUSSION THREAD

**Discussion by zezaeoh** [actions/actions-runner-controller#500] [Questions] [answered]
Hi there,

i wonder how can i change update strategy of runner deployment resource.
i would like to change update strategy to "recreate" to use persistent volume.

is there any good way to use persistent volume?

**Comment by mumoshu** [comment, marked as answer]
@zezaeoh Hey! Unfortunately no. I believe we need a concept like `volumeClaimTemplates` in the first place to use PV in any meaningful way. Otherwise, your max runner replica is limited to only 1, which doesn't seem practical to me.

**Comment by mumoshu** [comment]
BTW, it would be nicer to use Discussions for questions!

Issues are for bug reports and feature requests.

**Reply by zezaeoh** [reply]
thanks a lot :)
i'll keep in mind

**Comment by samus-aran** [comment]
We're locking this discussion because it has not had recent activity and/or other members have asked for more information to assist you but received no response. Thank you for helping us maintain a productive and tidy community for all our members.
